SECURITY.md

```diff
@@ -8,7 +8,7 @@ as possible.
 
 Please provide a clear and concise description of the vulnerability, including:
 
-* the affected version(s) of Open-Source-Office,
+* the affected version(s) of ouroboros-network,
```

Comment:
Should it be _the affected version(s) of all packages included in ouroboros-network repository_?

Reply:
Yes, can resend with it saying " the affected version(s) of all packages included in ouroboros-network-repository"

Reply:
Updated text. @coot